src/platforms/node/guides/express/index.mdx

```diff
@@ -63,7 +63,7 @@ app.get("/debug-sentry", function mainHandler(req, res) {
 
 Possible options are:
 
-```js
+```JavaScript
 // keys to be extracted from req
 request?: boolean | string[]; // default: true = ['cookies', 'data', 'headers', 'method', 'query_string', 'url']
 // server name
```

Comment:
```suggestion

// server name
```

I'd do the rest here but they're not close enough to what you changed for it to be willing to let me make suggestions there. Personally I think spacing it out makes it much easier to read, but you'll have to do it locally if you want to do it.

Reply:
Done.